includes/results.inc

```diff
@@ -1021,7 +1021,11 @@ class IslandoraSolrFacets {
       $link['count'] = $count;
       $link['link_plus'] = '<a' . drupal_attributes($attributes['plus']['attr']) . '>+</a>';
       $link['link_minus'] = '<a' . drupal_attributes($attributes['minus']['attr']) . '>-</a>';
-      $buckets[] = $link;
+      $buckets[$bucket] = $link;
+    }
+
+    if (isset($this->settings['solr_field_settings']['sort_by']) && $this->settings['solr_field_settings']['sort_by'] == 'index') {
+      ksort($buckets);
```

Comment:
@adam-vessey Just re-reading your comments
> For example, say you're limiting the number of values returned, and by numbers, would only get things near the bottom of things lexicographically... Could end up with strange results.

This is totally true, but all I am providing is a way to re-order the facets you have from by _count_ to by _value_. I was trying not to sell this as anything more than adjusting the sort order of your existing facets.